lib/stockpile.rb

```diff
@@ -122,6 +123,25 @@ def perform_cached(db: :default, key:, ttl: Stockpile::DEFAULT_TTL, &block)
     )
   end
 
+  # Attempt to reset the TTL for a value cached under the given key.
+  #
+  # @param key [String] Key to renew cache TTL on
+  # @param db [Symbol] (optional) Which Redis database to cache data in.
+  #   Defaults to `:default`
+  # @param ttl [Integer] (optional) Time in seconds to expire cache after.
+  #   Defaults to Stockpile::DEFAULT_TTL
+  #
+  # @yield [block] A block of code to be executed in case of cache miss
+  #
+  # @example Renew cache operation
+  #   Stockpile.renew_cache(key: 'meaning_of_life', ttl: 42)
+  #
+  # @return [true] if the key existed (and was successfully renewed)
+  # @return [false] if the key did not exist
+  def renew_cache(key:, db: :default, ttl: Stockpile::DEFAULT_TTL)
+    Stockpile::CachedValueRenewer.renew(key: key, db: db, ttl: ttl)
```

Comment:
Can you explain where `renew` is defined? I see `Stockpile::CachedValueRenewer` implements `renew_cached` but not `renew`.

Reply:
Great catch, yeah I totally typoed the name there. Fixed.